### DISCUSSION TITLE
How to use the not expression???

### DISCUSSION THREAD

**Discussion by LGSIMM** [datalust/seq-tickets#1943] [Help] [answered]
I'm using this which works and shows results.
@Properties['x-app-version'] = 'Release-1114'

But if I want to find all logs where the release is **NOT** 1114, I thought this would work.
@Properties['x-app-version'] not 'Release-1114'

But it doesn't.
I know there are results in the logs which should be shown.
How do I do this correctly?

**Comment by liammclennan** [comment, marked as answer]
Hi @LGSIMM ,

`not` is logical negation. What you want is the inequality comparison operator `<>`. 

`@Properties['x-app-version'] <> 'Release-1114'`

**Reply by LGSIMM** [reply]
I tried a few other things as well, like:

@Properties['x-app-version'] < 'Release-1114'

I thought this would show less than, but it doesn't work either.
So I have to use <> to show all results less than or more than.

Thanks, that works ok.

**Reply by liammclennan** [reply]
`<>` is 'not equal to', the opposite of `=`. It is not the same as `<` and `>` operators. 

You could also use `not(@Properties['Action'] = 'Create')` which has a slightly different meaning.
